server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -627,3 +627,61 @@ pub fn router() -> Router {
             .route("attempt/msg/:msg_id/", get(list_attempts_by_msg)),
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use validator::Validate;
+
+    use super::{ListAttemptedMessagesQueryParameters, ListAttemptsByEndpointQueryParameters};
+
+    #[test]
+    fn test_list_attempted_messages_query_params_validation() {
+        let q: ListAttemptedMessagesQueryParameters = serde_json::from_str(
+            &r#"{
+                "channel": ",invalid",
+                "status": 3
+            }"#,
+        )
+        .unwrap();
+        assert!(q.validate().is_err());
+
+        let q: ListAttemptedMessagesQueryParameters = serde_json::from_str(
+            &r#"{
+                "channel": "valid",
+                "status": 0
+            }"#,
+        )
+        .unwrap();
+        q.validate().unwrap();
+    }
+
+    #[test]
+    fn test_list_attempts_by_endpoint_query_parameters_validation() {
+        let q: ListAttemptsByEndpointQueryParameters = serde_json::from_str(
+            &r#"{
+                "event_types": [ "valid_event_type_name", ",invalid_event_type_name"],
+                "channel": "valid-channel-name"
+            }"#,
+        )
+        .unwrap();
+        assert!(q.validate().is_err());
+
+        let q: ListAttemptsByEndpointQueryParameters = serde_json::from_str(
+            &r#"{
+                "event_types": [ "valid-event-type-name", "another-valid-event-type-name"],
+                "channel": ",invalid-channel-name"
+            }"#,
+        )
+        .unwrap();
+        assert!(q.validate().is_err());
+
+        let q: ListAttemptsByEndpointQueryParameters = serde_json::from_str(
+            &r#"{
+                "event_types": [ "valid-event-type-name", "another-valid-event-type-name"],
```

Comment:
Whitespace is inconsistent.
```suggestion
                "event_types": ["valid-event-type-name", "another-valid-event-type-name"],
```